geoportailv3/static/js/maincontroller.js

```diff
@@ -920,4 +925,13 @@ app.MainController.prototype.toggleTiles3dVisibility = function() {
   }
 };
 
+/**
+ * Check if disconnected or offline mode enabled.
+ * @returns {boolean}
+ * @export
+ */
+app.MainController.prototype.isOfflineState = function() {
+  return this.offlineMode.isEnabled() || !!this.networkStatus_.offline;
```

Comment:
In ngeo, the networkStatus could be clarified and use a getter: `this.networkStatus_.isDisconnected()`.